tests/test_repository_lib.py

```diff
@@ -236,31 +236,31 @@ def test_get_metadata_fileinfo(self):
       file_object.write('test file')
 
     # Generate test fileinfo object.  It is assumed SHA256 and SHA512 hashes
-    # are computed by get_metadata_fileinfo().
+    # are computed by get_targets_metadata_fileinfo().
     file_length = os.path.getsize(test_filepath)
     sha256_digest_object = securesystemslib.hash.digest_filename(test_filepath)
     sha512_digest_object = securesystemslib.hash.digest_filename(test_filepath, algorithm='sha512')
     file_hashes = {'sha256': sha256_digest_object.hexdigest(),
                    'sha512': sha512_digest_object.hexdigest()}
     fileinfo = {'length': file_length, 'hashes': file_hashes}
-    self.assertTrue(tuf.formats.FILEINFO_SCHEMA.matches(fileinfo))
+    self.assertTrue(tuf.formats.TARGETS_FILEINFO_SCHEMA.matches(fileinfo))
 
     storage_backend = securesystemslib.storage.FilesystemBackend()
 
-    self.assertEqual(fileinfo, repo_lib.get_metadata_fileinfo(test_filepath,
+    self.assertEqual(fileinfo, repo_lib.get_targets_metadata_fileinfo(test_filepath,
                                                               storage_backend))
```

Comment:
Nit: this indentation looks odd, to maintain the existing style of the surrounding code (which, admittedly, contradicts the coding style) it should align with `test_filepath` above
```suggestion
                                                                      storage_backend))
```